src/App.js

```diff
@@ -1,32 +1,35 @@
 import React from 'react';
 import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
-import SearchResults from './context/SearchResults';
```

Comment:
Why have you renamed this file?😄. I thought it contained search logic?😄.

Reply:
Also, there's been so much shifting.😄. I think we could work more on making the app architecture easy to navigate and manage.

Reply:
@katia-utochkina : Because we needed separate responsibilities between managing our map center and locations for our map.